src/main/java/net/sf/jabref/model/groups/SearchGroup.java

```diff
@@ -151,42 +159,13 @@ public boolean isDynamic() {
     }
 
     @Override
-    public String getDescription() {
-        return this.query.getDescription();
-    }
-
```

Comment:
They were clearly not used. :-) May be a bug, but also a problem when moving them as they would introduce a dependency on logic in model. Better to reintroduce the functionality properly later. (And if you look at the translation strings only the one for "All entries" was actually removed, so everything else is still there (and the full description of "All entries" is never shown as it is only shown when you edit the group and you can't edit that group.)